src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -27,8 +27,55 @@
 namespace Microsoft.ML.Trainers
 {
 
-    /// <include file='doc.xml' path='doc/members/member[@name="LBFGS"]/*' />
-    /// <include file='doc.xml' path='docs/members/example[@name="LogisticRegressionBinaryClassifier"]/*' />
+    /// <summary>
+    /// The <see cref="IEstimator{TTransformer}"/> to predict a target using a linear logistic regression model trained with L-BFGS method.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this trainer, use [LbfgsLogisticRegression](xref:Microsoft.ML.StandardTrainersCatalog.LbfgsLogisticRegression(BinaryClassificationCatalog.BinaryClassificationTrainers, string, string, string, float, float, float, int, bool))
```

Comment:
>LbfgsLogisticRegression(BinaryClassificationCatalog.BinaryClassificationTrainers, string, string, string, float, float, float, int, bool)) [](start = 104, length = 138)

this UID won't work, you need fully qualified names. please use [https://xref.docs.microsoft.com/autocomplete?text=LbfgsLogisticRegression](https://xref.docs.microsoft.com/autocomplete?text=LbfgsLogisticRegression)  to get the UID. It's this one:

Microsoft.ML.StandardTrainersCatalog.LbfgsLogisticRegression(Microsoft.ML.BinaryClassificationCatalog.BinaryClassificationTrainers,System.String,System.String,System.String,System.Single,System.Single,System.Single,System.Int32,System.Boolean)

please also fix for the other one.

Reply:
Thanks. Sure.

---
In reply to: [276468758](https://github.com/dotnet/machinelearning/pull/3385#discussion_r276468758) [](ancestors = 276468758)